src/components/organelle-wrapper.tsx

```diff
@@ -10,58 +10,197 @@ interface OrganelleWrapper {
-      calculatedProperties: {},
-      clickHandlers: [
-        {
-          selector: '#melanocyte_x5F_cell',
-          action: this.organelleClick.bind(this, 'cytoplasm')
+      calculatedProperties: {
+        saturation: {
+          ratio: {
+            numerator: {
+              count: {
+                species: 'melanosome',
+                state: [
+                  'waiting_on_actin_terminal',
+                  'waiting_on_nuclear_actin_terminal'
+                ],
+              }
+            },
+            denominator: 20
+          }
         },
+        lightness: {
+          ratio: {
+            numerator: {
+              count: {
+                species: 'melanosome',
+                state: 'waiting_on_nuclear_actin_terminal'
+              }
+            },
+            denominator: 10
+          }
+        },
+        grayness: {
+          ratio: {
+            numerator: {
+              count: {
+                species: 'melanosome',
+                state: [
+                  'waiting_on_actin_terminal',
+                  'waiting_on_nuclear_actin_terminal'
+                ],
+                rules: {
+                  fact: 'size',
+                  greaterThan: 0.7
+                }
+              }
+            },
+            denominator: {
+              count: {
+                species: 'melanosome',
+                state: ['waiting_on_actin_terminal', 'waiting_on_nuclear_actin_terminal'],
+                rules: {
+                  fact: 'size',
+                  lessThan: 0.7
+                }
+              }
+            }
+          }
+        }
+      },
+      clickHandlers: [
+        // {
+        //   selector: this.organelleInfo.cytoplasm.selector,
+        //   action: this.organelleClick.bind(this, 'cytoplasm')
+        // },
         {
-          selector: '#nucleus_x5F_A',
+          selector: this.organelleInfo.nucleus.selector,
           action: this.organelleClick.bind(this, 'nucleus')
         },
+        {
+          selector: this.organelleInfo.golgi.selector,
+          action: this.organelleClick.bind(this, 'golgi')
+        }
       ],
       species: [
+        'organelles/melanosome.yml',
         'organelles/hexagon.yml',
-        'organelles/triangle.yml',
-        'organelles/g-protein.yml',
-        'organelles/g-protein-part.yml'
-      ]
+        'organelles/triangle.yml'
+        // 'organelles/g-protein.yml',
+        // 'organelles/g-protein-part.yml'
+      ],
+      hotStart: 1000
     }).then((m: any) => {
       this.model = m;
+      this.completeLoad();
+    });
+  }
 
-      this.model.setTimeout(() => {
-        let cell: any = document.querySelector(`#${this.props.name} #cellshape_0_Layer0_0_FILL`);
-        if (cell) {
-            cell.style['fill-opacity'] = 0.5;
-            cell.style.fill = 'rgb(241,212,151)';
-        }
-      },                    10);
+  completeLoad() {
+    this.model.on('model.step', () => {
```

Comment:
To confirm, is this the fix for situations where the SVG was modified before it was fully loaded?